src/scenes/Event/WithdrawingTopic/Reward.js

```diff
@@ -0,0 +1,131 @@
+  const qtumReturnRate = totalBetAmount ? ((qtumWinnings - totalBetAmount) / totalBetAmount) * 100 : 0;
+  const botReturnRate = totalVoteAmount ? ((botWinnings - totalVoteAmount) / totalVoteAmount) * 100 : 0;
+  // const invalidOption = localizeInvalidOption(topic.options[topic.resultIdx], intl);
+  const resultBetAmount = betBalances[topic.resultIdx];
+  const resultVoteAmount = voteBalances[topic.resultIdx];
+  const totalQtumWinningBets = eventPage.topic.qtumAmount[topic.resultIdx];
+  const totalQtumBets = _.sum(topic.qtumAmount);
+  const totalLosingQtumBets = totalQtumBets - totalQtumWinningBets;
+  const losersQtumReward = totalLosingQtumBets / 100;
+  const losersAdjustedQtum = totalLosingQtumBets - losersQtumReward;
+  const qtumWon = ((resultBetAmount / totalQtumWinningBets) * losersAdjustedQtum) || 0;
+  const totalBotWinningBets = topic.botAmount[topic.resultIdx];
+  const botQtumWon = ((resultVoteAmount / totalBotWinningBets) * losersQtumReward) || 0;
+  if (botQtumWon > 0 || qtumWon > 0) {
+    return (
+      <Container>
+        <RewardIcon />
+        <RewardTitle />
+        <Row>
+          <QtumReturn
+            qtumWinnings={eventPage.qtumWinnings}
+            qtumWon={qtumWon}
+            botQtumWon={botQtumWon}
+            resultTokenAmount={resultBetAmount}
+            totalTokenAmount={totalBetAmount}
+            tokenWinnings={qtumWinnings}
+            qtumReturnRate={qtumReturnRate}
+          />
+          <Separator />
+          <BotUsed
+            botWinnings={eventPage.botWinnings}
+            resultTokenAmount={resultVoteAmount}
+            totalTokenAmount={totalVoteAmount}
+            tokenWinnings={botWinnings}
+            botReturnRate={botReturnRate}
+          />
+        </Row>
+      </Container>
+    );
+  }
+  return <Fragment />;
+};
+
+const RewardIcon = () => <Icon type='token' />;
+
+const RewardTitle = () => (
+  <Label>
+    <FormattedMessage id="withdrawDetail.reward" defaultMessage="REWARD">
+      {(txt) => i18nToUpperCase(txt)}
+    </FormattedMessage>
+  </Label>
+);
+
+const Separator = styled.div`
+  display: inline-block;
+  width: 1px;
+  height: 75px;
+  background: ${props => props.theme.palette.divider};
+  margin-left: ${props => props.theme.padding.md.px};
+  margin-right: ${props => props.theme.padding.md.px};
+`;
+
+const QtumReturn = injectIntl(({ qtumWinnings, qtumReturnRate, intl, ...props }) => (
+  <Wrapper>
+    <Typography variant="display1">
+      <Row>
+        +{qtumWinnings} <Token>QTUM</Token>
+        <Tooltip id="tooltip-reward" title={<RewardTooltipContent token="QTUM" {...props} />}>
+          <i className="icon iconfont icon-ic_question" />
+        </Tooltip>
+      </Row>
+    </Typography>
+    <Typography variant="caption">
+      {`${intl.formatMessage({ id: 'withdrawDetail.returnRate' })} ${qtumReturnRate.toFixed(2)}%`}
+    </Typography>
+  </Wrapper>
+));
+
+const BotUsed = injectIntl(({ botWinnings, botReturnRate, intl, ...props }) => (
```

Comment:
`BotReturn` to match the other component.

Reply:
miss this, will include in the next PR, I plan to update the sorting features